proxy/src/main/java/io/openlineage/proxy/api/models/HttpConfig.java

```diff
@@ -0,0 +1,19 @@
+/*
+/* Copyright 2018-2022 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.proxy.api.models;
+
+import io.openlineage.proxy.ProxyStreamConfig;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@NoArgsConstructor
+@ToString
+public final class HttpConfig implements ProxyStreamConfig {
+  @Getter @Setter private String url;
+  @Getter @Setter private String apikey;
```

Comment:
_Minor_: I would use camel case for `apiKey` as that aligns with our name convention for our clients, etc.

Reply:
good point